.travis.yml

```diff
@@ -13,15 +13,13 @@ matrix:
     - php: 7.3
     - php: 7.4
 
-sudo: false
-
 cache:
   directories:
     - vendor
     - $HOME/.composer/cache
 
 install:
-  - composer install --no-interaction --prefer-source
+  - composer install --no-interaction --prefer-dist
```

Comment:
Please, can you remove option `--prefer-source/dist`? `--prefer-dist` is default in Composer, via [doc](https://getcomposer.org/doc/03-cli.md#install-i)

Reply:
Done.